module/VuFind/src/VuFind/Recommend/LibGuidesProfile.php

```diff
@@ -79,6 +107,14 @@ public function __construct(
 
         // Cache the data related to profiles for up to 10 minutes:
         $this->cacheLifetime = intval($config->GetAccounts->cache_lifetime ?? 600);
+
+        $profile = $config->Profile;
+        if ($profile) {
+            $this->strategies = $profile->get('strategies', []);
```

Comment:
Hmm, the problem is $config->get doesn't return an array normally, it's some object.  The default value [] works if there is no config present, but I'd prefer it as the object otherwise.

Reply:
Good point. The behavior of the Config object is annoying just about as often as it's helpful. :-) I think this still leaves us with the problem that `$this->strategies` may end up being set to a string if the configuration isn't quite right. Maybe something like:

```
$strategies = $profile->get('strategies', []);
$this->strategies = is_string($strategies) ? [$strategies] : $strategies;
```

Reply:
Yup that works.